corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/util/util.js

```diff
@@ -0,0 +1,24 @@
+function Util() {
+}
+
+Util.getQueryParams = function (qs) {
+    qs = qs.split("+").join(" ");
+
+    var params = [], tokens,
+        re = /[?&]?([^=]+)=([^&]*)/g;
+
+    while (tokens = re.exec(qs)) {
```

Comment:
FWIW, I feel strongly against turning that off wholesale (because `if (foo = 5)` is an easy mistake to make), but am on board with turning it off for the individual line (because I think the pattern Will's using is fine, possibly because it's common in perl).